package.json

```diff
@@ -9,31 +9,34 @@
     ]
   },
   "scripts": {
-    "account": "yarn workspace @se-2/hardhat account",
-    "chain": "yarn workspace @se-2/hardhat chain",
-    "fork": "yarn workspace @se-2/hardhat fork",
-    "deploy": "yarn workspace @se-2/hardhat deploy",
-    "verify": "yarn workspace @se-2/hardhat verify",
-    "compile": "yarn workspace @se-2/hardhat compile",
-    "generate": "yarn workspace @se-2/hardhat generate",
-    "hardhat:lint": "yarn workspace @se-2/hardhat lint",
-    "hardhat:lint-staged": "yarn workspace @se-2/hardhat lint-staged",
-    "hardhat:test": "yarn workspace @se-2/hardhat test",
-    "start": "yarn workspace @se-2/nextjs dev",
-    "next:lint": "yarn workspace @se-2/nextjs lint",
-    "next:format": "yarn workspace @se-2/nextjs format",
-    "next:check-types": "yarn workspace @se-2/nextjs check-types",
+    "account": "pnpm --filter @se-2/hardhat account",
+    "chain": "pnpm --filter @se-2/hardhat chain",
+    "fork": "pnpm --filter @se-2/hardhat fork",
+    "hardhat:deploy": "pnpm --filter @se-2/hardhat hardhat:deploy",
```

Comment:
Why do we need the prefix `hardhat` here?

I think we want to keep chain / deploy / start as it was.

Reply:
Oh, I read the description now... sorry. haha

That is going to be a though one....